src/main/groovy/com/marklogic/gradle/task/NewProjectTask.groovy

```diff
@@ -36,7 +36,8 @@ class NewProjectTask extends MarkLogicTask {
 			ant.input(message: "Test REST API port (intended for running automated tests; leave blank for no server):", addproperty: "mlTestRestPort")
 		}
 		ant.input(message: "Do you want support for multiple environments? ", validargs: "y,n", addproperty: "mlPropertiesPlugin", defaultvalue: "y")
-		ant.input(message: "Do you want resource files for a content database and set of users/roles created?", validargs: "y,n", addproperty: "mlScaffold", defaultvalue: "y")
+		ant.input(message: "Do you want resource files for a content database?", validargs: "y,n", addproperty: "mlScaffoldDatabases", defaultvalue: "y")
+		ant.input(message: "Do you want a set of users/roles created?", validargs: "y,n", addproperty: "mlScaffoldSecurity", defaultvalue: "y")
```

Comment:
This is a better property name - mlScaffoldSecurity - keep that instead of "mlSecurity" even after removing the database question.